<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-95104MM" failures="1" tests="2" name="LoginTest" time="11.864" errors="0" timestamp="2021-09-24T13:42:04 IST">
  <testcase classname="Automation.Salesforce.LoginTest" name="loginTest" time="2.859"/>
  <testcase classname="Automation.Salesforce.LoginTest" name="getTextStatus" time="0.793">
    <failure type="java.lang.AssertionError" message="expected [a customer] but found []">
      <![CDATA[java.lang.AssertionError: expected [a customer] but found []
at Automation.Salesforce.LoginTest.getTextStatus(LoginTest.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- getTextStatus -->
</testsuite> <!-- LoginTest -->
